pkg/lint/rules/template.go

```diff
@@ -101,7 +101,7 @@ func Templates(linter *support.Linter, values []byte, namespace string, strict b
 		linter.RunLinterRule(support.ErrorSev, path, validateAllowedExtension(fileName))
 
 		// We only apply the following lint rules to yaml files
-		if filepath.Ext(fileName) != ".yaml" {
+		if filepath.Ext(fileName) != ".yaml" || filepath.Ext(fileName) == ".yml" {
```

Comment:
Shouldn't this be `!=  ".yml"`?

Reply:
Good catch. In fact, it should be:

`filepath.Ext(fileName) != ".yaml" && filepath.Ext(fileName) != ".yml"`

as we only want to go to `continue` if it's not `.yaml` and it's not `.yml`.